copycat.tmux

```diff
@@ -6,6 +6,22 @@ source "$CURRENT_DIR/scripts/variables.sh"
 source "$CURRENT_DIR/scripts/helpers.sh"
 source "$CURRENT_DIR/scripts/stored_search_helpers.sh"
 
+# check if tmux supports keytable (v2.1+)
+check_keytable_support() {
+	if (( $(echo "$(tmux -V | cut -f2 -d' ') >= 2.1" | bc) )); then
+		keytable_supported=true
+	fi
+}
```

Comment:
Hey, we already have the script for checking tmux version, check usage [here](https://github.com/tmux-plugins/tmux-copycat/blob/33b98809a5b46fecc8bb48f0e3ccd801689981e4/scripts/copycat_mode_start.sh#L10).

Also, let's use `$VERSION_WITH_KEY_TABLE_SUPPORT` (or similar) instead of the raw `2.1` number.